core/src/smartcontracts/isi/mod.rs

```diff
@@ -485,23 +482,4 @@ mod tests {
         ));
         Ok(())
     }
-
-    #[test]
-    async fn transaction_signed_by_genesis_account_should_be_rejected() -> Result<()> {
```

Comment:
This check was removed from `accept` function.

Reply:
@mversic any opinion on this one?

Reply:
I can't recall why I removed it

Reply:
I think it should be returned

Reply:
returned